src/sysupdate/sysupdated.c

```diff
@@ -1397,7 +1568,7 @@ static const sd_bus_vtable target_vtable[] = {
 
         SD_BUS_METHOD_WITH_ARGS("Vacuum",
                                 SD_BUS_NO_ARGS,
-                                SD_BUS_RESULT("u", count),
+                                SD_BUS_RESULT("u", instances, "u", disabled_transfers),
```

Comment:
i guess this is not really a compat break, given we haven#t released this yet hence ok

Reply:
yep. Part of the reason this is an RC blocker :)